Modules/Log/Get-SysmonNetwork.ps1

```diff
@@ -0,0 +1,84 @@
+<#
+.SYNOPSIS
+    Get-SysmonNetwork.ps1 extracts all Sysmon Network Events [Evt 3] from the Sysymon Operational Event log for a specified timeframe
+.DESCRIPTION
+    Query the event log and pull back all Sysmon Process Creation events. Configured for Sysmon 5.02
+    Event 3
+    Query and filter
+.PARAMETER
+    Switch to pull back Network events back a desired number of minutes
+    [int32]$BackMins=180. Defaults to 180 minutes = 3 hours
+    Some time guides: 180 = 3 hours, 360 = 6hours, 720 = 12 hours, 1440 = 1 day, 2880 = 2 days, 4320 = 3 days, 10080 = 7 days
+    Keep in mind depending on sysmon configuration, the longer the timeframe the more work to pull events.
+.EXAMPLE
+    .\Get-SysmonNetwork.ps1 -BackMins 180
+    .\Get-SysmonNetwork.ps1 180
+    .\Get-SysmonNetwork.ps1
+
+    HostName            : Win10.ACME.local
+    Version             : 5
+    EventType           : Network connection detected
+    EventID             : 3
+    DateUTC             : 2017-01-18T02:16:53
+    ProcessGuid         : b7480112-13e6-587c-0000-00109a016400
+    ProcessId           : 4572
+    Image               : C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
+    User                : ACME\USER1
+    Protocol            : tcp
+    Initiated           : True
+    SourceIsIpv6        : False
+    SourceIp            : 10.1.1.21
+    SourceHostname      : Win10.ACME.local
+    SourcePort          : 55047
+    SourcePortName      : 
+    DestinationIsIpv6   : False
+    DestinationIp       : 216.58.220.131
+    DestinationHostname : syd09s01-in-f131.1e100.net
+    DestinationPort     : 443
+    DestinationPortName : https
+
+.LINK
+.NOTES
+OUTPUT tsv
```

Comment:
The OUTPUT directives are no longer needed and no longer work, per http://trustedsignal.blogspot.com/2015/08/kansas-stafford-release-more-capable.html, see the part about the -OutputFormat parameter.

Reply:
SOrry - just noticed this comment. 
I have removed the -OUTPUT directive. Overall the version of sysmon this was written to is a bit dated, the fields extracted for this content should still be valid.